website/components/PluginSupportStatusTable.module.scss

```
@@ -29,6 +29,16 @@
   }
 }
 
+.PluginLink {
+  color: var(--rp-c-brand);
```

Add font-weight: 500 to align with Rspress links.

```suggestion
  color: var(--rp-c-brand);
  font-weight: 500;
```